### PR TITLE
Add fix for resolving inspection error on bundled lightning libraries

### PR DESCRIPTION
**Pull request by jbstocker**

### CODE REVIEW

File: devtools/lightning-inspect.js [hunk 1]
```diff
@@ -221,15 +221,15 @@ window.attachInspector = function({Application, Element, ElementCore, Stage, Com
         oSetParent.apply(this, arguments);
 
         if (!window.mutatingChildren) {
-            if (parent) {
+            if (parent && parent.dhtml) {
```
Comment (netjlr) [inline devtools/lightning-inspect.js:224]: Without the check for parent.dhtml, the code can get to line 227 and throw the error below:

lightning-inspect.js:227 Uncaught (in promise) TypeError: parent.dhtml is not a function
    at StateMachineRouter.Element._setParent (lightning-inspect.js:227)
    at ElementChildList._connectParent (ElementChildList.mjs:50)

File: devtools/lightning-inspect.js [hunk 2]
```diff
@@ -268,9 +268,13 @@ window.attachInspector = function({Application, Element, ElementCore, Stage, Com
             c = c._element;
         }
         if (v == dv) {
-            c.dhtmlRemoveAttribute(n);
+            if (c.dhtmlRemoveAttribute) {
```
Comment (netjlr) [inline devtools/lightning-inspect.js:271]: Same logic as with parent.dhtml; in some of my testing the code was getting to line 271 with an error that c object didn't have a .dhtmlRemoveAttribute function; adding this second if-check fixes that.

**Comment by wouterlucas** [comment]
Looks same as https://github.com/rdkcentral/Lightning/pull/201 ?

**Comment by netjlr** [comment]
> Looks same as #201 ?

I agree. @jbstocker

**Comment by jbstocker** [comment]
> Looks same as #201 ?

Not identical but yes it's clearly aiming to resolve same issue.  As long as the issue is resolved I don't mind which PR survives haha.

**Comment by erikhaandrikman** [comment]
I suggest merging this PR since it ship some extra checks for `dhtmlSetAttribute` and `dhtmlRemoveAttribute` on the Element's prototype

**Review by erikhaandrikman — APPROVED** [review]

**Pull request merged**